test/long/org/apache/cassandra/cql3/CorruptionTest.java

```diff
@@ -145,10 +145,10 @@ private void dumpKeys(byte[] putdata, byte[] getdata) throws IOException {
                     String basename = "bad-data-tid" + Thread.currentThread().getId();
                     File put = new File(basename+"-put");
                     File get = new File(basename+"-get");
-                    try(FileWriter pw = new FileWriter(put.toJavaIOFile())) {
+                    try(FileWriter pw = put.newWriter(File.WriteMode.OVERWRITE)) {
```

Comment:
Formatting: While we are here can we put the brace in the next line?

Reply:
done